libyul/AsmParser.cpp

```diff
@@ -345,6 +337,9 @@ Parser::ElementaryOperation Parser::parseElementaryOperation()
 		ret = std::move(literal);
 		break;
 	}
+	case Token::HexStringLiteral:
+		fatalParserError(3772_error, "Hex literals are not valid in this context.");
```

Comment:
Why not? The changes in `Token.h` say it's valid in Yul.

Reply:
It is valid in the object parser. I want to merge `AsmParser` and `ObjectParser` in the future.

Reply:
Ok